package.json

```diff
@@ -10,11 +10,16 @@
     "wu": "^2.1.0"
   },
   "devDependencies": {
+    "babel": "^6.5.2",
+    "babel-cli": "^6.10.1",
```

Comment:
I don't think you need `babel-cli`. Just `babel-core`, `babel-loader`, `babel-polyfill`, `babel-preset-es2015`.